packages/terra-app-delegate/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Changelog
 Unreleased
 ----------
 
+### Added
+* Added registerDismissLock
+
+### Changed
+* Updated README to reflect new API
```

Comment:
Will this need to be a major version bump?

Reply:
The change itself is passive. I'm leaving the old APIs in place, and the DisclosureManager will handle the old stuff appropriately. I'm content with marking things as deprecated and removing at them at the next-next version bump (after the React 16 bump). Or we can turn around immediately after React 16 is released and bump things again.